manim/utils/space_ops.py

```diff
@@ -582,19 +527,22 @@ def line_intersection(line1: Sequence[float], line2: Sequence[float]) -> np.ndar
     ValueError
         Error is produced if the two lines don't intersect with each other
     """
-    x_diff = (line1[0][0] - line1[1][0], line2[0][0] - line2[1][0])
-    y_diff = (line1[0][1] - line1[1][1], line2[0][1] - line2[1][1])
-
-    def det(a, b):
-        return a[0] * b[1] - a[1] * b[0]
-
-    div = det(x_diff, y_diff)
-    if div == 0:
-        raise ValueError("Lines do not intersect")
-    d = (det(*line1), det(*line2))
-    x = det(d, x_diff) / div
-    y = det(d, y_diff) / div
-    return np.array([x, y, 0])
+    if any(np.array([line1, line2])[:, :, 2].reshape(-1)):
+        # checks for z coordinates != 0
+        raise ValueError("Intersection for coords in the xy-plane.")
```

Comment:
```suggestion
        raise ValueError("Coords must be in the x-y plane.")
```
I think this sounds better